### PR TITLE
Added network capture capability to test-configuration-options.md

### PR DESCRIPTION
**Pull request by kristofmuhi**
Updated the appium capability to the correct one:

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by kristofmuhi** [comment]
@kimsauce  is this page for appium or sauceCTL or both?

Because the capability for Appium is: 
`sauceLabsNetworkCaptureEnabled`

And for saucectl
networkCapture

**Comment by kimsauce** [comment]
> @kimsauce is this page for appium or sauceCTL or both?
> 
> Because the capability for Appium is: `sauceLabsNetworkCaptureEnabled`
> 
> And for saucectl networkCapture

Hi @kristofmuhi ! Pls see comment here: https://docs.google.com/document/d/1xEv4hSSJ9cC_TeXfkU_uTRE2684Wso9wsTnDMYl7Too/edit?usp=sharing.

**Comment by wswebcreation** [comment]
> @kimsauce is this page for appium or sauceCTL or both?
> 
> Because the capability for Appium is: `sauceLabsNetworkCaptureEnabled`
> 
> And for saucectl networkCapture

@kristofmuhi 

Why do we add `sauceLabs` to the cap? It's part of the `sauce:options`-object which makes the prefix `sauceLabs` not needed and a bit redundant for all Sauce LaBs specific options

**Comment by wswebcreation** [comment]
Then maybe update the PR 😉

File: docs/dev/test-configuration-options.md [hunk 1]
```diff
@@ -598,7 +598,7 @@ Enables the interception of biometric input, allowing the test to simulate Touch
 Enables the use of the app's private app container directory instead of the shared app group container directory. For testing on the Real Device Cloud, the app gets resigned, which is why the shared directory is not accessible.
 
 ---
-### `networkCapture`
+### `sauceLabsNetworkCaptureEnabled`
```
Comment (kimsauce) [inline docs/dev/test-configuration-options.md:601]: @kristofmuhi to confirm, this one is for Appium? And `networkCapture` is for Espresso/XCUITest?

**Review by kimsauce — CHANGES_REQUESTED** [review]
Please update these docs with the networkCapture capability:

https://docs.saucelabs.com/mobile-apps/automated-testing/espresso-xcuitest/espresso/
https://docs.saucelabs.com/mobile-apps/automated-testing/espresso-xcuitest/xcuitest/

File: docs/dev/test-configuration-options.md (hunk 1 above)
Comment (kristofmuhi) [reply docs/dev/test-configuration-options.md:601]: nope. Appium is changed to networkCapture. 
so it is consistent through all platforms and frameworks!

Comment (kimsauce) [reply docs/dev/test-configuration-options.md:601]: Thank you for clarifying! Could you or @aoleszyn pls update the PR?

**Review by lysannep — APPROVED** [review]

**Pull request merged**